qiskit_nature/second_q/operators/sparse_label_op.py

```diff
@@ -409,16 +426,16 @@ def chop(self, tol: float | None = None) -> SparseLabelOp:
 
         new_data = {}
         for key, value in self.items():
-            zero_real = cmath.isclose(value.real, 0.0, abs_tol=tol)
-            zero_imag = cmath.isclose(value.imag, 0.0, abs_tol=tol)
-            if zero_real and zero_imag:
-                continue
-            if zero_imag:
-                new_data[key] = value.real
-            elif zero_real:
-                new_data[key] = value.imag
-            else:
-                new_data[key] = value
+            if not isinstance(value, ParameterExpression):
```

Comment:
This `isinstance` check is a bit too agressive. I expect the following to print `True` which it does not currently:
```python
from qiskit.circuit import Parameter
from qiskit_nature.second_q.operators import FermionicOp

a = Parameter("a")
op1 = FermionicOp({"+_0": a})
print((op1 - op1).chop() == FermionicOp.zero())  # False
```